content/docs/contributing/style_guide.md

```diff
@@ -32,23 +32,53 @@ When it is advantageous, data should be [stored in data files and processed usin
 
 ## Style Enforcement
 
-Markdown styling is enforced by [markdownlint](https://github.com/markdownlint/markdownlint) using the configuration {{% repo_link path=".mdl_style.rb" text=".mdl_style.rb" %}}.
-An explanation of the rules can be found [here](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md).
+Markdown styling is enforced by [`markdownlint`](https://github.com/markdownlint/markdownlint) using the configuration {{% repo_link path=".mdl_style.rb" text=".mdl_style.rb" %}}.
+An explanation of the style rules can be found [here](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md).
 
-You can optionally install markdownlint locally to check your changes before submitting a pull request.
+You can optionally install `markdownlint` locally to check your changes before submitting a pull request.
+
+### Using `pre-commit`
+
+The easiest way to do this is to use the [pre-commit](https://pre-commit.com/) tool, which has already been set up in the handbook repository.
+
+```bash
+brew install pre-commit   # if not already installed
+pre-commit run markdownlint --all-files
+```
+
+`pre-commit` is intended to be run before all commits (to make sure that you don't unintentionally commit something unwanted).
+To see how to set this up, see the [Contributing Changes]({{< ref "/content/docs/contributing/contributing_changes.md#making-changes" >}}) page.
+
+### Manual Installation
```

Comment:
I'd personally suggest removing this **Manual Installation** section entirely now. My feeling is that it adds complexity, without adding much benefit given that `pre-commit run markdownlint --all-files` achieves much the same.

Reply:
I was thinking about that, too.

Reply:
(But let's wait until the `pre-commit` setup is definitely sorted out. 😄)